google-userinfo/background/main.js

```diff
@@ -0,0 +1,23 @@
+
+function notifyUser(user) {
+  browser.notifications.create({
+    "type": "basic",
+    "title": "Google info",
+    "message": `Hi ${user.name}`
+  });}
+
+function logError(error) {
+  console.error(`Error: ${error}`);
+}
+
+/*
+When the button's clicked:
+- get an access token using the identity API
+- use it to get the user's info
+- show a notification containing some of it
+*/
+browser.browserAction.onClicked.addListener(() => {
```

Comment:
If CLIENT_ID hasn't been changed, and the user hasn't gone through the setup step, would it make sense to error here?